web/controllers/user_controller.ex

```diff
@@ -0,0 +1,29 @@
+defmodule UeberauthProvider.UserController do
+  use UeberauthProvider.Web, :controller
+
+  alias UeberauthProvider.User
+
+  def index(conn, params) do
+    users = Repo.all(User)
+    render conn, "index.html", users: users
+  end
+
+  def new(conn, params) do
```

Comment:
doesn't that forget previously entered values in the form? So if there's a re-render because of a validation error all the fields would be blank?